### PR TITLE
Minor revisions

### PR DESCRIPTION
**Pull request by KristjanESPERANTO**
This should have no functional impact. This PR is mostly the part of PR #157 that has nothing to do with the upgrade to hafas-client v6.

### CODE REVIEW

File: BvgFetcher.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
 const createClient = require("hafas-client");
 const shortenStationName = require("vbb-short-station-name");
-const bvgProfile = require("hafas-client/p/bvg");
+const profile = require("hafas-client/p/bvg");
```
Comment (rejas) [inline BvgFetcher.js:3]: i'd still call it bvgProfile since it is one :-)

**Review by rejas — APPROVED** [review]

**Comment by rejas** [comment]
just one nitpick, if you want I can merge it anyway :-)

File: BvgFetcher.js (hunk 1 above)
Comment (KristjanESPERANTO) [reply BvgFetcher.js:3]: To better compare the code to the MMM-PublicTransportHafas fork, I'd like a more generic name.
Would ‘hafasProfil‘ be an acceptable compromise?

Comment (rejas) [reply BvgFetcher.js:3]: Not needed, I understand and will merge

**Pull request merged**